decidim-proposals/app/forms/decidim/proposals/proposal_form.rb

```diff
@@ -16,10 +16,9 @@ class ProposalForm < Decidim::Form
       attribute :user_group_id, Integer
       attribute :has_address, Boolean
       attribute :attachment, AttachmentForm
-
       validates :title, :body, presence: true, etiquette: true
       validates :title, length: { maximum: 150 }
-      validates :body, length: { maximum: 500 }, etiquette: true
```

Comment:
Can you readd the `etiquette: true` part, please?

`validates :body, etiquette: true`

Reply:
I think that this isn't necessary, we already have this two lines before: 
>         validates :title, :body, presence: true, etiquette: true

Reply:
Oh, you're right! 💪